EIPS/eip-5450.md

```diff
@@ -62,12 +65,10 @@ The time and memory complexity if this pass is *O(len(code))*.
 #### Operand stack validation
 
 In the second validation phase data-flow (TODO: or control-flow) analysis is performed on the code.
-For each reachable instruction in the code the operand stack height is recorded.
+For each instruction in the code the operand stack height is recorded.
 The first instruction has the recorded stack height equal to the number of inputs to the function matching the code.
 
-The FIFO queue *worklist* is used for tracking "to be inspected" instructions. Initially, it contains the first instruction.
-
-While *worklist* is not empty, pop an instruction and:
+The instructions are processed in-order until there are no more instructions.
 
 1. Determine the effect the instruction has on the operand stack:
```

Comment:
It should also reset current recorded stack height after terminating instructions and RJUMP.